processing/src/main/java/org/apache/druid/segment/column/ColumnCapabilitiesImpl.java

```diff
@@ -177,6 +177,21 @@ public static ColumnCapabilitiesImpl createSimpleNumericColumnCapabilities(Value
     return builder;
   }
 
+  /**
+   * Simple, non dictionary encoded string without bitmap index or anything fancy
+   */
+  public static ColumnCapabilitiesImpl createSimpleStringColumnCapabilities()
+  {
+    return new ColumnCapabilitiesImpl().setType(ValueType.STRING)
+                                       .setHasMultipleValues(false)
```

Comment:
Is it ok that hasMultipleValues is `false` here? That seems like the potentially-sketchiest of the behaviors, since someone might use this method for a multi-value string accidentally. How about naming it `createSimpleSingleValueStringColumnCapabilities`?

Method names: the longer the better. Go, Java.

Reply:
renamed ^